enterprise/setup/kubernetes_amazon_eks.mdx

```diff
@@ -0,0 +1,430 @@
+* **AWS_KMS_LOGS_CMK**: `arn:aws:kms:REGION:ACCOUNTID:key/KEYID`<br />
+  This needs to match the `Arn` value of the Amazon KMS Key
+* **AWS_REGION**: `REGION`<br />
+  Replace this with the AWS region you have deployed to (e.g. `us-west-1`)
+* **alb.ingress.kubernetes.io/inbound-cidrs**: `0.0.0.0/0`<br />
+  Adjust this to restrict the allowed source IPs that can access the load balancer
+* **alb.ingress.kubernetes.io/scheme**:<br />
+  Set this to either `internet-facing` to give the load balancer a public IP, or `internal` for internal access only
+
+Now deploy the pganalyze Enterprise application:
+
+```
+kubectl apply -f pganalyze-enterprise.yml
+```
+
+We can confirm the deployment is completed by checking `kubectl get deploy`:
+
+```
+$ kubectl get deploy
+NAME        READY   UP-TO-DATE   AVAILABLE   AGE
+pganalyze   1/1     1            1           1m
+```
+
+### Step 6: Run the Enterprise self check to verify the configuration and license
+
+Run the following command to perform the Enterprise self-check:
+
+```
+kubectl exec -i -t deploy/pganalyze -- /bin/bash -c "source /etc/profile.d/rvm.sh && SECRET_KEY_BASE=1 bundle exec rake enterprise:self_check"
+```
+
+This should return the following:
+
+```
+Testing database connection... Success!
+Testing Redis connection... Success!
+Skipping SMTP mailer check - configure MAILER_URL to enable mail sending
+Verifying enterprise license... Success!
+
+All tests completed successfully!
+```
+
+If you see an error, double check your configuration settings, and especially the database connections.
+
+In case you get an error for the license verification, please reach out to the pganalyze team.
+
+### Step 7: Initialize the database
+
+Run the following to initialize the pganalyze statistics database:
+
+```
+kubectl exec -i -t deploy/pganalyze -- /bin/bash -c "source /etc/profile.d/rvm.sh && SECRET_KEY_BASE=1 bundle exec rake db:setup"
+```
+
+This will also output the admin credentials like this:
+
+```
+Database 'postgres' already exists
+ set_config 
+------------
+ 
+(1 row)
+
+INFO -- : *****************************
+INFO -- : *** INITIAL ADMIN CREATED ***
+INFO -- : *****************************
+INFO -- : 
+INFO -- : *****************************
+INFO -- : Email:    admin@example.com
+INFO -- : Password: PASSWORDPASSWORD
+INFO -- : *****************************
+INFO -- : 
+INFO -- : Use these credentials to login and then change email address and password.
+```
+
+Now we can connect to the pganalyze UI. Run the following to determine the hostname of the load balancer that was provisioned:
+
+```
+$ kubectl get ingress
+```
+
+```
+NAME        CLASS    HOSTS   ADDRESS                                        PORTS   AGE
+pganalyze   <none>   *       HOSTNAME.REGION.elb.amazonaws.com              80      5m58s
+```
+
+When you go to the hostname in your browser you should see the login page. You can now use the initial admin details to log in:
+
+![](login.png)
```

Comment:
Are these images already here? I don't see them in the PR.

Reply:
This is repurposing the images from the existing VM-based guide (since this part of it works pretty much the same, and we don't have a shared explanation for that yet)